src/team-card/TeamCard.js

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import Profile from "./Profile";
+import {
+  createTheme,
+  MantineProvider,
+  Title,
+  SimpleGrid,
+  Container,
+} from "@mantine/core";
+import { useMediaQuery } from "@mantine/hooks";
+import "@mantine/core/styles.css";
+import "./TeamCard.css";
+
+const theme = createTheme({
+  breakpoints: {
+    xs: "10em",
+  },
+});
+
+function TeamCard({ title, team_members }) {
+  const small = useMediaQuery("(max-width: 500px)");
+  const titleSize = small ? "20px" : "30px";
+
+  return (
+    <MantineProvider theme={theme}>
```

Comment:
We wanna use MantineProvider to wrap around only the root component, (so that every child then is technically wrapped around MantineProvider). Try removing MantineProvider and the theme, and to prevent SimpleGrid from reducing to 1 column, add the `base` key to cols: `cols={{ lg: 6, sm: 4, xs: 2, base: 2 }}` on line 32.

<img width="630" alt="mantine'd" src="https://github.com/uoftblueprint/website-v2/assets/81433191/0d5ee618-bf82-407c-adc1-954303c3e168">

Reply:
Ooh wait that makes more sense, thanks!! 👍